### DISCUSSION TITLE
Installation error Linux Mint 22.4. Version 3.4.12

### DISCUSSION THREAD

**Discussion by vdijken** [pawelsalawa/sqlitestudio#5181] [General]
Gives installation error as local user in every directory
![Schermafdruk van 2024-12-26 10-59-24](https://github.com/user-attachments/assets/b442aaa8-1aa7-4fe6-908d-3a20e82b38e9)
; see attachement

**Comment by pawelsalawa** [comment]
If you run `logname` in your terminal - do you get results, or does it say the 'command not found' error?

**Reply by pawelsalawa** [reply]
Okay. I see now that this problem was solved in master branch, but it was not back-ported to 3.4.x line: https://github.com/pawelsalawa/sqlitestudio/issues/4828 (https://github.com/pawelsalawa/sqlitestudio/commit/5c28dd8266e483df6435ec69d0941f6d57543632)

I've created #5182 to track it.
